.php_cs.cache

```diff
@@ -0,0 +1 @@
+{"php":"7.2.10-0ubuntu0.18.04.1","version":"2.14.0:v2.14.0#b788ea0af899cedc8114dca7db119c93b6685da2","rules":{"blank_line_after_namespace":true,"braces":true,"class_definition":true,"elseif":true,"function_declaration":true,"indentation_type":true,"line_ending":true,"lowercase_constants":true,"lowercase_keywords":true,"method_argument_space":{"on_multiline":"ensure_fully_multiline"},"no_break_comment":true,"no_closing_tag":true,"no_spaces_after_function_name":true,"no_spaces_inside_parenthesis":true,"no_trailing_whitespace":true,"no_trailing_whitespace_in_comment":true,"single_blank_line_at_eof":true,"single_class_element_per_statement":{"elements":["property"]},"single_import_per_statement":true,"single_line_after_imports":true,"switch_case_semicolon_to_colon":true,"switch_case_space":true,"visibility_required":true,"encoding":true,"full_opening_tag":true},"hashes":{"src\/Robots.php":3165021838,"src\/RobotsMeta.php":2924386571,"src\/RobotsTxt.php":2893133003,"src\/RobotsHeaders.php":444848160}}
```

Comment:
This doesn't need to be committed please remove this.

Reply:
Done. 

Why this file isn't in the .gitignore ?